src/flat-world.js

```diff
@@ -101,7 +101,8 @@ const flatWorld = module.exports = {
   },
 
   getPixelRatio: function getPixelRatio(ratio) {
```

Comment:
I assume we are making this change because of your previous fix to support retina?

What's the long-term situation here, do we no longer need this function? Should we remove the parameter?